Project.toml

```
@@ -40,6 +41,7 @@ PrettyTables = "0.9, 0.10, 0.11"
 RecipesBase = "0.7, 0.8, 1.0"
 SpecialFunctions = "^0.8, 0.9, 0.10, 1.0"
 StatsBase = "0.32, 0.33"
+StatsModelComparisons = "0.1.0"
```

To avoid pulling in too many dependencies:
```suggestion
StatsModelComparisons = "0.1.1"
```